examples/synthea-breast-cancer/Ali_Laiba_syntheaEDA/SyntheaClassifiers.ipynb

```diff
@@ -0,0 +1,1710 @@
+    "\n",
+    "dt_pred_no_scale = tree_classifier_no_scale.predict(X_test_no_scale)\n",
+    "print_results(y_test_no_scale, dt_pred_no_scale)\n",
+    "\n",
+    "tree.export_graphviz(tree_classifier_no_scale, out_file='cancer_model.dot',\n",
+    "                     feature_names=['# of Surgeries','# of Radiation Therapies','Cancer Improved?', 'Diagnosis Age', \n",
+    "                                    'Primary Tumour Category', 'Regional Node Category', '# of Medications'],\n",
+    "                     class_names=sorted(['Stage 1', 'Stage 2', 'Stage 3', 'Stage 4']), label='all', rounded=True, filled=True)\n",
+    "\n",
+    "create_png = subprocess.run([\"dot -Tpng cancer_model.dot -o cancer_model.png\"], shell=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Decision Tree Visualization](cancer_model.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='DTAnalysis'></a>\n",
+    "**Analysing our Results**\n",
+    "\n",
+    "From the results above, we can see that our decision tree classifier is quite accurate, with a balanced accuracy figure of 0.9852. It also has high macro f1 score of 0.9830 and an MCC score of 0.9606. These scores depict the high precision and accuracy of the model, as well as its tendency not to produce very many false positives and negatives, which is corroborated by the confusion matrix. The model, as seen with the confusion matrix underpredicted the stage of cancer only once, and overpredicted six times, and even then, the predicted stage only ever deviated from the actual stage by one level. The lack of underpredictions is quite good, since, in our opinion, it is better to overpredict than underpredict as the impacts of an underprediction seem to be higher.\n",
+    "\n",
+    "The Decision Tree is quite visualizable, as can be seen from the tree diagram above, making the model quite good in helping us generate actionable insights from the model. Now I'm not a medical expert and so the insights generated are not fully interpretable by myself, but I can see that the primary tumour category has a significant bearing on the cancer stage, given that it's the root node. Interestingly, it seems like the number of radiation therapies, whether or not the cancer improved and the number of medications have no bearing on the stage which is not necessarily something that one would expect in real life. Perhaps this is due to the sparse quantities of data that were actually present for those pieces of data (eg. only 4.5% of cases present in the dataset had the patient's condition worsening).\n",
+    "\n",
+    "Overall, the Decision Tree Classifier was quite good in its predictions and it stacks up well against the other classifiers we ran as well, hence why it was one of our finalists for the prediction model."
```

Comment:
Really enjoyed your discussion of the Decision Tree classifier both here and in the Confluence discussion. Great analysis, choice, and justifications.